src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
+
   when (Done) {
     case Event(IssueReply, data) =>
+      isDone = true
```

Comment:
At some point you'll also need to remove the shutdown hook otherwise it has pointers to this actor and it can never be garbage collected